Cargo.toml

```diff
@@ -21,9 +21,41 @@ clippy.missing_const_for_fn = "deny"
 clippy.too_long_first_doc_paragraph = "warn"
 
 [workspace.dependencies]
-
-[profile.test.package.proptest]
-opt-level = 3
+async-trait = "0.1.83"
+bitcoin = { version = "0.32.5", features = [ # keep in sync with secp256k1
+  "serde",
+] }
```

Comment:
why you are bringing `bitcoin`, we don't need that. Or if it is for hashes we can use other rust crates that have less footprint.